unlock-app/src/components/content/CheckoutContent.tsx

```diff
@@ -6,22 +6,47 @@ import BrowserOnly from '../helpers/BrowserOnly'
 import Layout from '../interface/Layout'
 import Account from '../interface/Account'
 import { pageTitle } from '../../constants'
-import LogInSignUp from '../interface/LogInSignUp'
+import { Lock } from '../interface/checkout/Lock'
+import Loading from '../interface/Loading'
 import {
   Account as AccountType,
   Network,
   Router,
   PaywallConfig,
+  RawLock,
 } from '../../unlockTypes'
 import getConfigFromSearch from '../../utils/getConfigFromSearch'
+import { durationsAsTextFromSeconds } from '../../utils/durations'
+import { usePaywallLocks } from '../../hooks/usePaywallLocks'
 
 interface CheckoutContentProps {
   account: AccountType
   network: Network
   config?: PaywallConfig
 }
 
-export const CheckoutContent = ({ account, network }: CheckoutContentProps) => {
+const defaultLockAddresses: string[] = []
+
+const lockKeysAvailable = (lock: RawLock) => {
+  if ((lock as any).unlimitedKeys) {
```

Comment:
We're going to want to firm up the various lock types we have. Worse, it seems that Web3Service doesn't consistently return the same set of properties for all locks, which makes it difficult to reconcile these so there are some `any`s in this code.

Reply:
Yeah, Web3Service is due for a large refactor :/